### DISCUSSION TITLE
Tabview's close icon not working if calendar component is present in any of the tabs

### DISCUSSION THREAD

**Discussion by pvalecha94** [primefaces/community#1079] [PrimeNG] [answered]
Primeng tabview component's close icon is not working if any of the tabs is having primeng calendar component. pointer-events css property is set to none.
This is the stackblitz link depicting the above mentioned issue - (https://stackblitz.com/edit/h331qs?file=src%2Fapp%2Fdemo%2Ftab-view-closable-demo.html)
<img width="1727" alt="Screenshot 2024-02-02 at 11 35 11 AM" src="https://github.com/primefaces/community/assets/158470654/19f4688a-416b-4f95-abd2-ff4e89fec559">

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We've fixed the issue below which causes this behavior in 17.5.0, here is your example with v17.5.0 https://stackblitz.com/edit/h331qs-zl7ytt?file=src%2Fapp%2Fdemo%2Ftab-view-closable-demo.html 

- Calendar: Calendar CSS breaks other Components [\#14606](https://github.com/primefaces/primeng/issues/14606)
